python/pyspark/sql/dataframe.py

```diff
@@ -4570,7 +4570,9 @@ def subtract(self, other: "DataFrame") -> "DataFrame":
         ...
 
     @dispatch_df_method
-    def dropDuplicates(self, *subset: Union[str, List[str]]) -> "DataFrame":
+    def dropDuplicates(
+        self, subset: Optional[Union[str, List[str], _NoValueType]] = _NoValue, *subset_varargs: str
+    ) -> "DataFrame":
```

Comment:
Can we also update the docstring here for `subset` and add `subset_varargs`? Also add more examples?

Reply:
ah sure let me add them